### PR TITLE
Newt expansion Part 1

### PR DESCRIPTION
**Pull request by Scarlet-Oduko**
## About The Pull Request

Main Overview: Makes newt a playable job, along with making them a sub-type of human aka carbon. Lower health but faster then base speed, comes with inbuilt damage reduction for brute and burn since they won't be able to wear armor. Gear is restricted to outfit + they cannot shoot guns or operate any heavy weapons like a synth. 

Cannot be delimbed, has an innate ranged ability along with a healing ability due to pickup code being guh to work with.

Planned: Part two of this expansion will be focused on digsite upgrades for newt and digsite stuff in general to make them more worthwhile.
## Why It's Good For The Game

Newt always had some basic function as an ai companion, expanding on this to be fully playable, and with the recent researcher changes, I thought I might as well also put some more content to encourage them to be used, this is part one of an overall content expansion for researchers to pad out digsites.
## Changelog
:cl:
add: makes newt playable, new species that comes with innate weaknesses and buffs, newt has an innate ranged ability
imageadd: Sprites associated with newt and their outfit

/:cl:

### CODE REVIEW

**Comment by thatguythere03** [comment]
fitzshollsealofquality

**Comment by Scarlet-Oduko** [comment]
> fitzshollsealofquality

oh yeah, couldn't have done it without fitz, they provide all the sprites.

**Comment by Woker2004** [comment]
I do not know why I thought I was looking a Sybil github for a second, but I think it'd be a nice change for more variety, although that would make newt more up to date than AI, which is funny

**Comment by FosterGrime-Heymynameslouie** [comment]
wait a minute, would there be multiple newts since its now considered a job or would it only be locked to one?

also the thought of having a researcher and newt  combo ala a man and his dog is really cool
(dear God I jusr realized the kind of worse case scenario players whod main newt to be like)

**Comment by Scarlet-Oduko** [comment]
> wait a minute, would there be multiple newts since its now considered a job or would it only be locked to one?
> 
> also the thought of having a researcher and newt combo ala a man and his dog is really cool (dear God I jusr realized the kind of worse case scenario players whod main newt to be like)

It’s only locked to one, much like the FC and such.

**Comment by Scarlet-Oduko** [comment]
> I would make this so it's not combat focus but like roleplay focused if anything, similar to the one civie role or what not

All Corporate liaison does is shout propaganda, if people treat newt as a frontline dive xenos to get kills I’ll address it as I see it. Right now Newt is just speed and melee. Meant to help protect their savior the medical researcher.

they will be getting support stuff and all. They’re just a companion for researcher

**Comment by Woker2004** [comment]
> All Corporate liaison does is shout propaganda, if people treat newt as a frontline dive xenos to get kills I’ll address it as I see it. Right now Newt is just speed and melee. Meant to help protect their savior the medical researcher.
> 
> they will be getting support stuff and all. They’re just a companion for researcher

Just make newt only have like 5 melee damage, newt was added as a meme and should stay like one, though you could allow newt to understand xenos I guess? But if you want a fast melee person be bio light armor vali sword

**Comment by Scarlet-Oduko** [comment]
> > All Corporate liaison does is shout propaganda, if people treat newt as a frontline dive xenos to get kills I’ll address it as I see it. Right now Newt is just speed and melee. Meant to help protect their savior the medical researcher.
> > they will be getting support stuff and all. They’re just a companion for researcher
> 
> Just make newt only have like 5 melee damage, newt was added as a meme and should stay like one, though you could allow newt to understand xenos I guess? But if you want a fast melee person be bio light armor vali sword

Very weird reasoning, the whole point is to make newt something more then they were. Just because something’s a ‘meme’ doesn’t mean it has to stay one. Nor should it in my opinion

**Comment by Woker2004** [comment]
> the whole point is to make newt something more then they were. Just because something’s a ‘meme’ doesn’t mean it has to stay one. Nor should it in my opinion

Newt, was never spouse to even be combat based, he was just a meme to be there and ultimately just stay shipside, I'd just remove to combat aspect, we already have melee, that is more significant than what your planning here, also it should really stay as roleplay

**Comment by SzkieletorNutric** [comment]
Don't you fuckin' do it...

**Comment by Scarlet-Oduko** [comment]
> > the whole point is to make newt something more then they were. Just because something’s a ‘meme’ doesn’t mean it has to stay one. Nor should it in my opinion
> 
> Newt, was never spouse to even be combat based, he was just a meme to be there and ultimately just stay shipside, I'd just remove to combat aspect, we already have melee, that is more significant than what your planning here, also it should really stay as roleplay

Agree to disagree, concepts change, this is a tdm server. This is a flavor thing even if they’re combat capable. I have much planned and I don’t intend to stop unless a maintainer tells me to. I don’t care what’s seen as a meme. Quite frankly it’s better to make it more then a meme. We come for the wacky content by god I’m gonna give it to the people! Viva la newt.

**Comment by SzkieletorNutric** [comment]
> digsite stuff in general to make them more worthwhile

Digsites have always been more than worthwhile. The improvement they require instead is additional, unique and functional content in the form of implants and equipment that benefits the entire marine force, not gimmicks that are locked to a literal department pet and a role separate from the Researcher.

The bottom line is, when an admin takes over a pet just for the sake of pulling a joke and being a meme - it's actually funny and 'wacky'.
When you suggest to make an animal into a whole separate role and glue to it vague promises of functions that will potentially influence the overall balance of the game, it is anything but.

**Comment by Woker2004** [comment]
> When you suggest to make an animal into a whole separate role and glue to it vague promises of functions that will potentially influence the overall balance of the game, it is anything but.

Basically stole the words out of my mouth

**Comment by Woker2004** [comment]
Just not trying to be mean or anything, but odds are this will get closed when you PR it, maybe on another station like TG station or monkeystation wouldn't mind it, but as a Team Deathmatch server it kinda doesn't fit

**Comment by Scarlet-Oduko** [comment]
You misunderstand this then, this is a completely serious PR and not meant to be ‘haha newt’. If you’re not going to give proper feedback for what I’m doing and simply trying to bash the thing as a meme then I’m just not gonna continue this conversation. I have a lot planned. 

I even stated that I would be expanding digsites with not just newt content, So have a good day.

**Comment by Woker2004** [comment]
There really isn't much to say about it feedback wise, it's just a pet that stays shipside

**Comment by Scarlet-Oduko** [comment]
> Please add Gourmand as a playable variant.

That’ll most likely come in part two, I plan on making digsite rewards for both newt and marines, and the newt rewards will most likely be flavored around all the other cats like artificer getting bombs

**Comment by Ragecrabbo** [comment]
Neat idea though kind of a weird addition to the game. Hopefully this will see light of day although i wonder what their niche will be to seperate them from other support roles such as synth.

**Comment by Scarlet-Oduko** [comment]
> Neat idea though kind of a weird addition to the game. Hopefully this will see light of day although i wonder what their niche will be to seperate them from other support roles such as synth.

Right now there a little companion for researchers to help keep them safe/company when moving towards digsites. I plan on expanding researcher content as a whole once this is (hopefully) merged. Which will flesh out newts role based on equipment that will be able to be given to them. Based on the slugs from the game.

**Review by Kuro020 — APPROVED** [review]

**Review by RipGrayson — APPROVED** [review]

File: code/modules/mob/living/carbon/human/species.dm
Comment (Lumipharon) [inline code/modules/mob/living/carbon/human/species.dm:750]: less health but then they just have sythn dam reduction anyway to make up for it

File: code/modules/organs/limbs.dm
Comment (Lumipharon) [inline code/modules/organs/limbs.dm:279]: as before I don't see why this is needed

File: code/modules/power/cell.dm
Comment (Lumipharon) [inline code/modules/power/cell.dm:82]: stahp. I'm not going to mark every but if snowflake, you get the idea

File: code/datums/actions/species_actions/catslug_action.dm
Comment (Lumipharon) [inline code/datums/actions/species_actions/catslug_action.dm:32]: So you have a nodrop spear, and you don't actually throw you spear you just summon a magic projectile version?

**Review by Lumipharon — CHANGES_REQUESTED** [review]
there's uh, a lot of snowflake here

**Pull request closed without merging**
